apps/explorer/src/ui/TableCard.tsx

```diff
@@ -39,6 +40,7 @@ export function TableCard<DataType extends object>({
     columns,
     sortTable,
     defaultSorting,
+    fill = true,
```

Comment:
Don't default props to true, use `noFill` if this should be the default behavior. Also, this naming should be something like `fullWidth` not `fill`

Reply:
Updated to `noFullWidth`